ledger-core/v2/README.md

```diff
@@ -1,34 +1,49 @@
-[<img src="https://insolar.io/st/github-readme-banner.png">](http://insolar.io/?utm_source=Github)
+[<img src="https://github.com/insolar/doc-pics/raw/master/st/github-readme-banner.png">](http://insolar.io/?utm_source=Github)
 
-Insolar platform is the most secure, scalable, and comprehensive business-ready blockchain toolkit in the world. Insolar’s goal is to give businesses access to features and services that enable them to launch new decentralized applications quickly and easily. Whether a minimum viable product or full-scale production software, Insolar builds and integrates applications for your enterprise's existing systems.
+# Insolar Assured Ledger
 
-# Quick start
+## Introduction
+
+Insolar Assured Ledger is the most secure, scalable, and comprehensive business-ready blockchain platform.
+
+It provides:
+
+* Environment for running native Insolar and 3rd party applications.
 
 To learn what distinguishes Insolar from other blockchain projects, go through the [list of our features](https://insolar.io/platform?utm_source=Github). 
 
 To get a grip on how Insolar works, take a look at its [architecture overview](https://docs.insolar.io/en/latest/architecture.html#architecture).
 
+## Quick start
+
 To join the Insolar network, download the [latest release](https://github.com/insolar/assured-ledger/ledger-core/v2/releases) and follow the [integration instructions](https://docs.insolar.io/en/latest/integration.html).
 
-To test Insolar locally, install it and deploy as described below.
+You can test Assured Ledger locally:
 
-## Install
+1. Install everything from the **Prerequisites** section.
+2. Install this platform.
+3. Deploy it locally.
+4. Run tests
 
-1. Install the latest 1.14 version of the [Golang programming tools](https://golang.org/doc/install#install). Make sure the `$GOPATH` environment variable is set.
+### Prerequisites
 
-2. Download the Insolar package:
+Install v1.14 version of the [Golang programming tools](https://golang.org/doc/install#install). Set the [$GOPATH environment variable](https://github.com/golang/go/wiki/SettingGOPATH).
+
+### Install
+
+1. Download the Assured Ledger package:
 
    ```
    go get github.com/insolar/assured-ledger/ledger-core/v2
    ```
 
-3. Go to the package directory:
+2. Go to the package directory:
 
    ```
    cd $GOPATH/src/github.com/insolar/assured-ledger/ledger-core/v2
    ```
 
-4. Install dependencies and build binaries:
+3. Install dependencies and build binaries using [the makefile](https://github.com/insolar/assured-ledger/blob/PLAT-182-all-insolard-cli-commands/ledger-core/v2/Makefile) that automates this process:
```

Comment:
I wonder do we still need this step? Go will download all necessary packages during build step, make vendor is not needed.

Reply:
@sexyhamster *make vendor* step is still needed